phpBB/includes/acp/acp_database.php

```diff
@@ -219,8 +277,24 @@ function main($id, $mode)
 						{
 							if (confirm_box(true))
 							{
-								unlink($file_name);
-								$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_DB_DELETE');
+								try
+								{
+									// Delete from storage
+									$storage->delete($file_name);
+
+									// Add log entry
+									$phpbb_log->add('admin', $user->data['user_id'], $user->ip, 'LOG_DB_DELETE');
+
+									// Remove from database
+									$sql = "DELETE FROM " . $table_prefix . "backups
+										WHERE filename = '" . $file_name . "';";
```

Comment:
Please use $db->sql_escape() here

Reply:
Ok, i will, but the regexp doesn't allow any "dangerous" characters
https://github.com/rubencm/phpbb/blob/fa0cd7d1c6302413647f64008b55b8962e4e0c1c/phpBB/includes/acp/acp_database.php#L264
Also, the file name is generated by phpbb